apps/admin-gui/src/assets/config/defaultConfig.json

```
@@ -1,6 +1,6 @@
 {
   "config": "default",
-  "api_url": "http://localhost/krb/rpc",
+  "api_url": "https://login.cesnet.cz/oidc/",
```

```suggestion
  "api_url": "https://perun-dev.cesnet.cz/oauth/rpc/",
```
with what you typed it doesnt work